doc/rados/operations/index.rst

```diff
@@ -18,6 +18,7 @@ and, monitoring an operating cluster.
 	monitoring
 	monitoring-osd-pg
 	user-management
+    pg-repair
```

Comment:
Oh, it seems this got fixed in a different commit, since you went commit by commit you caught one that needs to be squashed!